frontend/src/metabase/sharing/components/AddEditSidebar.jsx

```diff
@@ -450,3 +486,66 @@ function formatDefaultParamValues(parameters, defaultParametersById) {
     })
     .filter(Boolean);
 }
+
+function ParametersSection({
```

Comment:
I think this should be moved to `metabase-enterprise`. Making it a plugin would let us remove `MetabaseSettings.isEnterprise()` and just use a single (possibly replaced) component.